R/Rules-methods.R

```diff
@@ -663,6 +663,39 @@ setMethod("nextBest",
 
 ## ============================================================
 
```

Comment:
sorry I forgot to mention this earlier, but can you please add documentation to this function, too? thx!